### DISCUSSION TITLE
Document update conflict

### DISCUSSION THREAD

**Discussion by kasgithub8** [Budibase/budibase#8244] [Help] [answered]
Hey,

adding and editing columns in the users table creates `Document update conflict.`
![image](https://user-images.githubusercontent.com/95284278/195600525-7b271fe0-3af8-47ff-a0e4-e8faea48aa7e.png)
Are there any workarounds? Can you manually update the users inside of couchdb?

**Comment by melohagan** [comment, marked as answer]
Hey @kasgithub8 

That looks like a bug. When did you notice that starting?

**Reply by kasgithub8** [reply]
Hey @melohagan,

I don't know when it started exactly but probably like 2-3 weeks ago was when I first noticed it.
Tbh, I haven't used it in so long but it could've started when 2.0 was announced.

**Reply by melohagan** [reply]
Is it always happening, or is it every so often? Self-hosted or cloud? 

I'm not able to recreate it myself. Perhaps your couchdb instance has gotten into a conflicted state: https://github.com/Budibase/budibase/discussions/7777#discussioncomment-3732390

**Reply by shogunpurple** [reply]
https://github.com/Budibase/budibase/issues/8043

**Reply by shogunpurple** [reply]
FIxed in latest

**Reply by melohagan** [reply]
Wonderful!

**Reply by kasgithub8** [reply]
When is the release of "latest"?

**Reply by shogunpurple** [reply]
I mean it's already released. The "latest" - as in the latest release

**Reply by melohagan** [reply]
Update to the *current* latest version - **v2.0.30**

**Reply by kasgithub8** [reply]
> I mean it's already released. The "latest" - as in the latest release

Alright, I'm gonna check out if the update is available tomorrow!

**Reply by melohagan** [reply]
It's available today!

**Reply by kasgithub8** [reply]
Yes, but im not in the office anymore. 😅

**Reply by melohagan** [reply]
Ah awesome haha!

**Reply by kasgithub8** [reply]
Thanks @melohagan and @shogunpurple.
It works!

**Reply by melohagan** [reply]
Brilliant! :)
